### ISSUE TITLE
Semver considerations

### ISSUE THREAD

**Issue by asbjornenge** [soyuka/ansi-rainbow#2]
Just a quick FYI; some of the changes introduced [here](https://github.com/soyuka/ansi-rainbow/commit/1dc9b4153dc8f9d06edf8981898913196588f252) - e.g. the use of `...colors` - broke a few of my modules (under node 4 LTS). May be a good idea to consider changes that require updating the version of node (comparet to the previously release), a `major` or at least `minor` change. :+1: 

Still, thanks for a great modules :smile:

**Comment by soyuka** [comment]
I did just that didn't I? 

Previous version was 0.0.9, the referred commit adds ([here](https://github.com/soyuka/ansi-rainbow/commit/1dc9b4153dc8f9d06edf8981898913196588f252#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R16)):

``` diff
+  "engines": {
+    "node": ">6.0"
+  },
```

Then, I released a major `1.0.0`. 

Or I totally failed because I released the major without publishing `1.0.0` to git but by publishing to npm only. What would explain the `1.0.1` just after the commit. 
In that's case, I'm really sorry about that :/. 

Solution: keep `0.x` for node < 4 or run the harmony flag with `1.x` ;)
